File: modulo1/segmento3/Apunte6/ejercicio2/ejercicio2.java/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
	public static void main(String args []) {
		Scanner keyboard = new Scanner(System.in);
			
		System.out.println("Para realizar la suma debe ingresar dos numeros");
		
		String salir = "n";


		System.out.print("Primer numero: ");
		Integer num1 = keyboard.nextInt();

		System.out.print("Segundo numero: ");
		Integer num2 = keyboard.nextInt();

		Integer res = num1 + num2;

		System.out.println("El resultado es: " +  res);

		System.out.println("¿Desea realizar otra suma? s/n");
		salir = keyboard.next();
		
		do {
			System.out.print("Primer numero: ");
			num1 = keyboard.nextInt();

			System.out.print("Segundo numero: ");
			num2 = keyboard.nextInt();

			res = num1 + num2;

			System.out.println("El resultado es: " + res);

			System.out.print("¿Desea realizar otra suma? s/n");
			salir = keyboard.next();
		} while(salir.equals("s"));
	}	
}

File: modulo1/segmento3/Apunte6/ejercicio2/ejercicio2.py
print("Para realizar la suma debe ingresar dos numeros")

num1 = int(input("Primer numero: "))
num2 = int(input("Segundo numero: "))

res = num1 + num2

print("El resultado es: ",  res)

salir = input("¿Desea continuar haciendo sumas? s/n: ")

while (salir != "n"):
	num1 = int(input("Primer numero: "))
	num2 = int(input("Segundo numero: "))
 
	res = num1 + num2
 
	print("El resultado es: ", res)

	salir = input("¿Desea continuar haciendo sumas? s/n: ")

